ee-9/feature-pack/src/main/resources/license/preview-feature-pack-licenses.xml

```diff
@@ -508,8 +508,8 @@
       </licenses>
     </dependency>
     <dependency>
-      <groupId>org.jboss.spec.javax.xml.soap</groupId>
-      <artifactId>jboss-saaj-api_1.4_spec</artifactId>
+      <groupId>org.jboss.spec.jakarta.xml.soap</groupId>
+      <artifactId>jboss-saaj-api_2.0_spec</artifactId>
```

Comment:
Please move this up into the right alphabetical order spot based on the new GA.

Reply:
Done.